compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs

```diff
@@ -282,6 +283,10 @@ data BuiltinExpr
   | BETrace                      -- :: forall a. Text -> a -> a
   | BEEqualContractId            -- :: forall a. ContractId a -> ContractId a -> Bool
   | BECoerceContractId           -- :: forall a b. ContractId a -> ContractId b
+
+  | BEEqualSerializable          -- :: ∀a. Serializable a -> a -> a -> Bool,
+  | BEIntIsSerializable            -- :: Serializable Int
+  | BEListIsSerializable           -- :: ∀a. Serializable a -> Serializable [a],
```

Comment:
How is this going to work for user-defined types, e.g., let’s say I define my own type isomorphic to the builtin list?

Reply:
See my first answer to @hurryabit